### PR TITLE
KafkaSink: Swap the default

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

- Reflect the change of the CE content mode based on https://github.com/knative-sandbox/eventing-kafka-broker/pull/2391

### CODE REVIEW

File: docs/eventing/sinks/kafka-sink.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ spec:
   # - structured
   # - binary
   #
-  # default: structured.
+  # default: binary.
```
Comment (pierDipi) [inline docs/eventing/sinks/kafka-sink.md:89]: Perhaps giving a primer on these 2 modes would be helpful as an addition in this page

**Review by pierDipi — APPROVED** [review]
/lgtm

File: docs/eventing/sinks/kafka-sink.md (hunk 1 above)
Comment (abrennan89) [reply docs/eventing/sinks/kafka-sink.md:89]: +1 @matzew can you provide some background information in a follow up issue or PR please

**Pull request merged**